Documentation/reproducible-research.rst

```diff
@@ -115,6 +115,14 @@ A non-exhaustive list is given below:
      - 1
      - `Taufiqurrahman et al. (2022) <https://doi.org/10.1002/essoar.10510965.1>`_
      - `zenodo.org/record/6386938 <https://zenodo.org/record/6386938>`_
+     
+   * - 2016 Mw 6.5 Norcia |br| earthquake
+     - dynamic
+     - N
+     - LSW
+     - 1
+     - `Tinti et al. (2021) <https://doi.org/10.1016/j.epsl.2021.117237>`_
+     - `github repository <https://github.com/git-taufiq/NorciaMultiFault>`_  
```

Comment:
Just a technical note: Maybe, we should link to the latest commit (https://github.com/git-taufiq/NorciaMultiFault/tree/6675035). After all git repositories can change...

Reply:
This comes along with the model in the paper published. I think no one will update it. Or we can put a release version?

Reply:
You can define a tag/release: https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository

Reply:
I would keep it as it is. If someone updates the repository, it means it is needed, and hopefully the new state is better than the older one.